apps/cookbook/src/app/showcase/default-list-showcase/default-list-showcase.html

```diff
@@ -0,0 +1,102 @@
+<p>
+  Default List adds styling to every <code>&lt;ul&gt;</code> and <code>&lt;ol&gt;</code> in Kirby.
+  It adds a padding of 4px to the left and a margin of 16px to the bottom of the entire
+  <code>&lt;ul&gt;</code> or <code>&lt;ol&gt;</code>, and a padding of 12px to the right of every
+  individual bullet or number in the <code>&lt;li&gt;</code> element(s). Lastly it adds a margin of
+  8px to the top of every <code>&lt;li&gt;</code>, except for the first
+  <code>&lt;li&gt;</code> element within the <code>&lt;ul&gt;</code> or <code>&lt;ol&gt;</code>.
+</p>
```

Comment:
I suggest changing the introductory paragraph to be a more general description of what default styles for lists Kirby are offering without mentioning specific sizes.
```suggestion
<p>
  Kirby offers default styling of ordered <code>&lt;ol&gt;</code> and unordered
  <code>&lt;ul&gt;</code> lists (but not description lists <code>&lt;dl&gt;</code>).
</p>
```